telegram-service/src/app.constants.ts

```diff
@@ -0,0 +1,16 @@
+export const enum SceneNames {
+    INPUT_CONTRACT_SCENE = 'INPUT_CONTRACT_SCENE'
+  };
+
+  export const enum Actions {
+    BID = 'bid',
+    SEND_TOKEN = 'send_token',
+    GO_BACK = 'go_back',
+  }
+
```

Comment:
please, configure eslint on your local folder, padding is off here and you cant see it, if you winn run `npm run format` prittier will fix the errors, but you should be able to see it in your IDE